pytorch_lightning/accelerators/accelerator.py

```diff
@@ -76,6 +76,8 @@ def auto_device_count() -> int:
     def is_available() -> bool:
         """Detect if the hardware is available."""
 
-    @classmethod
-    def register_accelerators(cls, accelerator_registry: Dict) -> None:
-        pass
+    @staticmethod
+    @abstractmethod
+    def name() -> str:
```

Comment:
The discussion for removing the `name` method. https://github.com/PyTorchLightning/pytorch-lightning/pull/12180#discussion_r818023970